Cubical/Data/FinSet/Examples.agda

```diff
@@ -0,0 +1,50 @@
+{-
+
+Computable stuff
+
+-}
+{-# OPTIONS --safe #-}
+
+module Cubical.Data.FinSet.Examples where
```

Comment:
This would fit better in the Experiments package

Reply:
Got it.